pkgs/data/fonts/sil-abyssinica/default.nix

```diff
@@ -0,0 +1,23 @@
+{ fetchzip, lib }:
+
+let
+  version = "1.500";
+in
+fetchzip {
+  name = "sil-abyssinica-${version}";
+  url = "mirror://debian/pool/main/f/fonts-sil-abyssinica/fonts-sil-abyssinica_${version}.orig.tar.xz";
+  sha256 = "sha256-fCa88wG2JfHTaHaBkuvoncbcbrh3XNzc8ewS3W+W/fM=";
+
+  postFetch = ''
+    mkdir -p $out/share/fonts
+    tar xf $downloadedFile --strip-components=1 -C $out/share/fonts AbyssinicaSIL-${version}/AbyssinicaSIL-R.ttf
+  '';
+
+  meta = with lib; {
+    description = "Unicode font for Ethiopian and Erythrean scripts (Amharic et al.)";
+    homepage = "http://scripts.sil.org/AbyssinicaSIL";
```

Comment:
```suggestion
    homepage = "https://software.sil.org/abyssinica/";
```

Reply:
Fixed, thanks heaps for the feedback and your help with my PRs, @SuperSandro2000, I really appreciate it